capture/http.c

```diff
@@ -201,6 +201,15 @@ unsigned char *moloch_http_send_sync(void *serverV, const char *method, const ch
         curl_easy_setopt(easy, CURLOPT_SSL_VERIFYHOST, 0L);
     }
 
+    // Send client certs if so configured
```

Comment:
The http code is used by other things then ES, so you'll need to add a moloch_http_set_client_cert (or whatever name) call kind of like moloch_http_set_retries and save it into the molochhttpserver_t and use that here.  Then in db.c you'll need to call it for the esServer

Reply:
Working on it... will rename the config variables to reflect the being ES only

Reply:
Done. Please let me know if this is what you had in mind.